packages/typespec-test/test/faceai/generated/typespec-ts/src/isUnexpected.ts

```diff
@@ -598,9 +663,15 @@ export function isUnexpected(
 export function isUnexpected(
   response:
     | AddPersonFace202Response
-    | AddPersonFaceFromUrl202Response
+    | AddPersonFaceLogicalResponse
```

Comment:
why this is missing before, I assume we don't add any changes related to lro logic response this time?

Reply:
bug here, see comments https://github.com/Azure/autorest.typescript/pull/2484#discussion_r1582501005